### ISSUE TITLE
Make arrow-parens rule more consistent

### ISSUE THREAD

**Issue by willclarktech** [LiskHQ/eslint-config-lisk-base#2]
Airbnb allows parentheses around a single argument in an arrow function to be dropped unless there's a block body. This seems inconsistent to me - I'd prefer being able to drop them in both cases. It would also mean less fiddling about with parentheses: I find myself switching between block/non-block body much more frequently than changing the number of arguments in a function.

http://eslint.org/docs/rules/arrow-parens
https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/es6.js#L24-L26

In other words:
```js
// good
const fn = foo => 'bar'; // as in airbnb
const fn = foo => {      // proposed change
  const bar = 'bar';
  return bar;
};

// bad
const fn = (foo) => 'bar'; // as in airbnb
const fn = (foo) => {      // proposed change
  const bar = 'bar';
  return bar;
};
```

**Comment by karmacoma** [comment]
@willclarktech Your proposal is fine by me. I agree being able to drop parens in both cases is better.
